mitiq/pec/representations/learning.py

```diff
@@ -64,24 +66,27 @@ def learn_depolarizing_noise_parameter(
             gates in each training circuit.
         training_random_state: Seed for sampling the training circuits.
         epsilon0: Initial guess for noise strength.
+        eta0: Initial guess for noise bias.
         observable (optional): Observable to compute the expectation value of.
             If None, the ``executor`` must return an expectation value.
             Otherwise the `QuantumResult` returned by `executor` is used to
             compute the expectation of the observable.
-        method: Type of optimizer. Must be an optimizer supported by
-            ``scipy.optimize.minimize``.
-        minimize_kwargs: Options to pass to the solver called by
-            ``scipy.optimize.minizmize``.
+        learning_kwargs (optional): Additional data and options including
+            ``pec_data`` from pre-executed runs of PEC on training circuits,
```

Comment:
What about if we don't mention `pec_data` at all in this docstring? 
It's a trick we use for CI testing but the user is not supposed to use that option, I suppose :-)

Note, that if the user is able to compute `pec_data` it means they know the full 2-by-2 parameter landscape and so a brute-force min of the loss is enough.

The real purpose of this function is to find a optimal solution by computing only a small path in the parameter landscape.

Reply:
I was undecided as to whether to mention it, so I've taken it out.